tools/wptserve/wptserve/sslutils/openssl.py

```diff
@@ -213,6 +218,7 @@ def get_config(root_dir, hosts, duration=30):
 subjectKeyIdentifier=hash
 authorityKeyIdentifier=keyid:always,issuer:always
 keyUsage = keyCertSign
+%(constraints_line)s
```

Comment:
Drive-by comment: This line above uses a variable that is local to the formatted string, which needs to be defined when formatting the string. This is named as the same variable as in the outer code block, but not implicitly linked.
To make this work, the in-line dictionary after the formatted string would need an additional key/value pair of

`"constraints_line": constraints_line,`